rhoas-cli-service-registry/README.adoc

```diff
@@ -243,40 +265,34 @@ Artifacts might include, for example, schemas that define the structure of Kafka
 [NOTE]
 ====
 You can use `rhoas service-registry use` to switch to a specific {registry} instance.
-
-.Selecting a {registry} instance to use
-[source,shell,subs="attributes"]
-----
-$ rhoas service-registry use --name=my-registry
-{registry} instance "my-registry" has been set as the current instance.
-----
 ====
 
 .Procedure
 
 . Upload a {registry} artifact.
 +
 --
-This example uploads a {registry} artifact called `my-artifact` to the {registry} instance.
+This example uploads {registry} artifact content called `my-artifact` to the {registry} instance.
 The artifact is an Apache Kafka Avro schema in JSON format.
 
-.Uploading an artifact
+.Uploading artifact content
 [source,shell]
 ----
 $ wget https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
---2021-09-27 16:17:18--  https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
```

Comment:
Line 267-277: Did you mean to remove this code entirely, or to move it into an “Example output” block?